pages/esimerkit.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: traffictype
+hero-image: code
+permalink: /ohjeita/esimerkit/
+section: Tietolähteet
+searchable: true
+lang: fi
+ref: code-examples
+title: Koodiesimerkit
+intro: Esimerkkejä Digitrafficin tarjoaman datan käytöstä ja hyvistä ohjelmointikäytännöistä
+---
+<div class="code-example">
+    <script src="https://gist.github.com/solita-ijunnone/c1b18cdadb1ac07a777f65f358c33d11.js"></script>
```

Comment:
Ois kyllä kiva, että organisation accountit tukisi myös gistejä, mutta nyt pitää mennä näin.